Q35/SMBus/smbus.inf

```diff
@@ -0,0 +1,47 @@
+;-------------------------------------------------------------------------------
+; smbusq35.INF
+;
+; Red Hat SM Bus Q35
+;
+; Copyright (c) 2017 Red Hat, Inc.  All rights reserved.
+;
+;-------------------------------------------------------------------------------
+
+[Version]
+Signature = "$Windows NT$"
+Class = System
+ClassGuid = {4D36E97D-E325-11CE-BFC1-08002BE10318}
+Provider = %RedHat%
+DriverVer=04/27/2017,100.0.0.0
+DriverPackageType = PlugAndPlay
+DriverPackageDisplayName = %smbus.DeviceDesc%
+CatalogFile  = smbus.cat
+
+; ======
+; Following two lines are questionable works without them
+DriverPackageType = PlugAndPlay
+DriverPackageDisplayName = %smbus.DeviceDesc%
+
+[Manufacturer]
+%smbus.DeviceDesc% = Models, NTamd64
```

Comment:
Yes it does, according to [this](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/inf-models-section):
> These platform extensions are not required in INF files for x86-based target operating system versions or non-PnP driver INF files.

However I can add it if you believe it is necessary.